chanjo/store/models.py

```diff
@@ -89,11 +87,13 @@ class TranscriptStat(BASE):
     threshold = Column(types.Integer)
     _incomplete_exons = Column(types.Text)
 
-    sample_id = Column(types.String(32), ForeignKey('sample.id'),
+    sample_id = Column(types.String(32), ForeignKey('sample.id', ondelete='CASCADE'),
```

Comment:
Same there, nice!

Reply:
without that param if you removed a sample it didin't remove the linked records in TranscriptStat any more. I guess something has changed in SQLAlchemy when switching to version 2?